### PR TITLE
Clustered RabbitMQ sample

### PR DESCRIPTION
**Pull request by andreasohlund**

### CODE REVIEW

File: samples/rabbitmq/cluster/sample.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+Note that [delayed delivery](/nservicebus/messaging/delayed-delivery.md) isn't supported for clusters and the sample therefor disables them. This means that [saga timeouts](/nservicebus/sagas/timeouts) can't be used and [delayed retries](https://docs.particular.net/nservicebus/recoverability.md#delayed-retries) must be disabled as shown below:
+
+snippet: cluster-disable-retries
+
+### Adding nodes
+
+RabbitMQ allows additional nodes to be added to avoid having to use a load balancer in front of the cluster. Adding those nodes will make the connection manager use them in a round robin fashion and automatically re-connect to healthy nodes should the connection be lost.
+
+Since RabbitMQ will default to port `5672` we only have to add nodes `rabbit2` and `rabbit3` as shown below:
```
Comment (WilliamBZA) [inline samples/rabbitmq/cluster/sample.md:50]:
```suggestion
RabbitMQ will default to port `5672` therefore the extra nodes (`rabbit2` and `rabbit3`) are added as shown below:
```

File: samples/rabbitmq/cluster/Rabbit_8/ClusteredRabbitMQ.sln.DotSettings [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```
Comment (timbussmann) [inline samples/rabbitmq/cluster/Rabbit_8/ClusteredRabbitMQ.sln.DotSettings:1]: don't commit the DotSettings file

File: samples/rabbitmq/cluster/Rabbit_8/MyEndpoint/Program.cs [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+            #endregion
+
+            endpointConfiguration.UseTransport(transport);
+            endpointConfiguration.EnableInstallers();
+
+            #region cluster-disable-retries
+            // Delayed retries needs to be disabled since we can't safely use the timeout capabilities of the transport
+            endpointConfiguration.Recoverability().Delayed(dc => dc.NumberOfRetries(0));
+            #endregion
+
```
Comment (timbussmann) [inline samples/rabbitmq/cluster/Rabbit_8/MyEndpoint/Program.cs:32]:
```suggestion
```

File: samples/rabbitmq/cluster/Rabbit_8/MyEndpoint/Program.cs [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+            var endpointInstance = await Endpoint.Start(endpointConfiguration)
+                .ConfigureAwait(false);
+
+            Console.WriteLine("Press S to send a message, any other key to exit");
```
Comment (timbussmann) [inline samples/rabbitmq/cluster/Rabbit_8/MyEndpoint/Program.cs:37]:
```suggestion
            Console.WriteLine("Press [s] to send a message, any other key to exit");
```

I kinda love how we use a different key for every sample 😆

File: samples/rabbitmq/cluster/Rabbit_8/MyEndpoint/Program.cs [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+            while (Console.ReadKey().KeyChar == 's')
```
Comment (timbussmann) [inline samples/rabbitmq/cluster/Rabbit_8/MyEndpoint/Program.cs:39]:
```suggestion
            while (Console.ReadKey().Key == ConsoleKey.S)
```

that one wouldn't be case sensitive

**Review by timbussmann — APPROVED** [review]

File: samples/rabbitmq/cluster/Rabbit_8/ClusteredRabbitMQ.sln.DotSettings (hunk 2 above)
Comment (andreasohlund) [reply samples/rabbitmq/cluster/Rabbit_8/ClusteredRabbitMQ.sln.DotSettings:1]: Hmm, the other samples seems to have them? https://github.com/Particular/docs.particular.net/blob/master/samples/rabbitmq/simple/Rabbit_7/SimpleRabbitMQ.sln.DotSettings

Comment (andreasohlund) [reply samples/rabbitmq/cluster/Rabbit_8/ClusteredRabbitMQ.sln.DotSettings]: Removed now

**Pull request merged**